### PR TITLE
Add code owners file, issue and pull request templates

### PR DESCRIPTION
**Pull request by fawadkhaliq**
**Description of changes**
- Added CODEOWNERS file to define individuals that are responsible for different charts
- Added issue templates for bug and feature request issues
- Added a checklist (based on commonly given feedback) to pull request template

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by fawadkhaliq** [comment]
CI should pass once https://github.com/aws/eks-charts/pull/416 is merged. Let's wait for https://github.com/aws/eks-charts/pull/416 to be merged before this one

**Review by kishorj — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
-Description of changes:
+<!-- Please link the GitHub issues related to this PR, if available -->
+
+### Description of changes
+
+<!-- Please explain the changes you made here. -->
+
+### Checklist
+- [ ] Added/modified documentation as required (such as the `README.md` for modified charts)
+- [ ] Bumped the chart `version` in `Chart.yaml` for the modified chart(s)
```
Comment (jaypipes) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: maybe change "Bumped" to "Incremented"? Is "bumped" widely understood, especially for ESL users?

**Review by jaypipes — APPROVED** [review]
:+1: badly needed, thank you @fawadkhaliq!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (fawadkhaliq) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: ++, will fix it. Thanks!

Comment (fawadkhaliq) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: Fixed: https://github.com/aws/eks-charts/pull/423

**Pull request merged**